core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -3317,6 +3317,16 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.MASTER)
           .build();
+  public static final PropertyKey MASTER_WORKER_DELETE_TIMEOUT_MS =
+      durationBuilder(Name.MASTER_WORKER_DELETE_TIMEOUT_MS)
+          .setAlias("alluxio.master.worker.delete.timeout.ms")
+          .setDefaultValue(Long.MAX_VALUE)
```

Comment:
i think the default can be something like `3 * MASTER_WORKER_TIMEOUT_MS`. @maobaolong By default do you prefer a lost worker to be kept in the `lost workers` forever, or removed from that list after some time?